src/main/java/com/spartasystems/holdmail/domain/Message.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright 2016 Sparta Systems, Inc
+ * Copyright 2016 - 2017 Sparta Systems, Inc
```

Comment:
Point taken.  There'll be more pull requests coming for this ticket, so will address then.